build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
```

Comment:
Sorry I didn't get your points.  My understanding:  GN will always tries to check the BUILD.gn file in src/ directory.
Only if it's not found, the build/secondary/BUILD.gn will be checked.  In our case, the src/BUILD.gn is always there..
Do you mean to add some options in command line like "gn gen out/Debug --root=/home/build --dotfile=/home/my_gn_file.gn"?
Another question is, should we enable build content_shell in the environment of both Crosswalk and Chromium-Crosswalk are downloaded?